openquake/qa_tests_data/classical/case_41/site_model.xml

```diff
@@ -4,7 +4,7 @@ xmlns="http://openquake.org/xmlns/nrml/0.5"
 xmlns:gml="http://www.opengis.net/gml"
 >
     <siteModel>
-        <site ec8="C" ec8_p18="D" h800="255.0" lat="-0.15738" lon="-64.98651" geology="CENOZOIC" slope="0.04" vs30="355.0" vs30Type="measured" z1pt0="408.4698342980704"/>
-        <site ec8="C" ec8_p18="D" h800="233.0" lat="-0.45686" lon="-64.77466" geology="PLEISTOCENE" slope="0.025" vs30="355.0" vs30Type="inferred" z1pt0="411.608194920163"/>
+        <site ec8="C" ec8_p18="D" h800="255.0" lat="-0.15738" lon="-64.98651" geology="CENOZOIC" slope="0.04" vs30="355.0" vs30Type="measured" z1pt0="408.4698342980704" eshm20_region="0"/>
+        <site ec8="C" ec8_p18="D" h800="233.0" lat="-0.45686" lon="-64.77466" geology="PLEISTOCENE" slope="0.025" vs30="355.0" vs30Type="inferred" z1pt0="411.608194920163" eshm20_region="0"/>
```

Comment:
Graeme, sorry I must have missed this in previous PRs. I was wondering if it wouldn't be better to change `eshm20_region` into `region`. More in general, I think that model parameters should be to the extent possible model-agnostic.

Reply:
Thanks Marco. I agree with your point, which is why this GMPE can be adapted in several different ways depending on how the user might envisage running the calculation. In this case, however, the `eshm20_region` refers to specific regions in Europe and the adaptations to the model they invoke cannot necessarily be applied anywhere outside these regions defined within this model. This is why I would prefer to stick with `eshm20_regions` rather than generalising to `regions`, as other regional adaptations for different GMPEs might take entirely different forms.